packages/web-components/fast-element/docs/api-report.md

```diff
@@ -387,6 +392,7 @@ export interface ExpressionObserver<TSource = any, TReturn = any, TParent = any>
     bind(controller: ExpressionController<TSource, TParent>): TReturn;
 }
 
+// Warning: (ae-forgotten-export) The symbol "FASTGlobal" needs to be exported by the entry point index.d.ts
```

Comment:
Hmmmm, is this type exposed by different a public export somehow? API extractor seems to think it should be exported.

Reply:
I can add this back. Not a biggie.